config/etc/system-image/writable-paths

```diff
@@ -19,7 +19,8 @@
 /var/snap                              auto                    persistent  transition  none
 /var/lib/bluetooth                      auto                    persistent  none        none
 # cloud-init
-/etc/cloud                              auto                    persistent  none        none
+/etc/cloud                              auto                    synced  none        none
+/etc/cloud/cloud.cfg.d                  auto                    synced  none        none
```

Comment:
this is redundant, only /etc/cloud will be enough... else you end up with stacked bind mounts